src/haddock/modules/topology/default.py

```diff
@@ -47,11 +77,13 @@ def run(self, module_information):
 
             # Split models
             logger.info(f"Split models if needed for {step_molecule_path}")
-            models = sorted(PDBFactory.split_ensemble(step_molecule_path))
+            ens = PDBFactory.split_ensemble(step_molecule_path)
```

Comment:
this is a very minor comment. But in general I would avoid creating this temporary variables. It creates confusion to the reader. If `ens` is not going to be used anywhere else, so use `sorted()` directly without creating the variable. If the temporary variable is really needed, use:

```python
_ = do_something()
real_variable = do_other(_)
```